website/docs/releases/1_24_2.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 1.24.2
+sidebar_position: 9933
+---
+
+# 1.24.2 - 2024-11-05
+
+### Added
+* **Spark: Add Dataproc run facet to include jobType property** [`#3167`](https://github.com/OpenLineage/OpenLineage/pull/3167) [@codelixir](https://github.com/codelixir)  
+    *Updates the GCP Dataproc run facet to include jobType property*
+* **Add EnvironmentVariablesRunFacet to core spec** [`#3186`](https://github.com/OpenLineage/OpenLineage/pull/3186) [@JDarDagran](https://github.com/JDarDagran)  
```

Comment:
```suggestion
* **Python: add EnvironmentVariablesRunFacet to core spec** [`#3186`](https://github.com/OpenLineage/OpenLineage/pull/3186) [@JDarDagran](https://github.com/JDarDagran)  
```

Reply:
Added description in subdescription since the spec part isn't Python specific